src/main/java/org/opentripplanner/middleware/controllers/api/AbstractUserController.java

```diff
@@ -43,8 +44,9 @@ protected void buildEndpoint(ApiEndpoint baseEndpoint) {
         ApiEndpoint modifiedEndpoint = baseEndpoint
             // Get user from token.
             .get(path(ROOT_ROUTE + TOKEN_PATH)
-                .withDescription("Retrieves an " + persistence.clazz.getSimpleName() + " entity using an Auth0 access token passed in an Authorization header.")
-                .withResponseType(persistence.clazz),
+                    .withDescription("Retrieves an " + persistence.clazz.getSimpleName() + " entity using an Auth0 access token passed in an Authorization header.")
+                    .withProduces(MIMETYPES_JSONONLY)
```

Comment:
Could we call this variable `JSON_ONLY_RESPONSE`?

Reply:
The `MIMETYPES_` prefix is to denote a list to avoid confusion. What do others think?

Reply:
I already added `withConsumes` to PUT and POST methods under the generic `ApiController`. The GET methods don't consume any request body so they don't have the `withConsumes` descriptor.

Reply:
I went ahead and renamed JSON_ONLY in https://github.com/ibi-group/otp-middleware/pull/55/commits/98227fcb96323ab7aca90adcd4d1d5599ad07942, it is more readable that way.